static/src/js/actions/__tests__/focusedCollection.test.js

```diff
@@ -161,8 +161,8 @@ describe('getFocusedCollection', () => {
         expect(getSearchGranulesMock).toHaveBeenCalledTimes(1)
       })
 
-      describe('when the requested collection is cwic and a polygon search is active', () => {
-        test('should toggle the polygon warning, update the focusedCollection and call getSearchGranules', async () => {
+      describe('when the requested collection is cwic and a polygon search is active and we try and retrieve a non existant gibs tag', () => {
+        test('Same test as above but no gibs tags, ensure it is not called', async () => {
```

Comment:
"Same test as above" is not a good descriptive name of what the test is doing. But I don't think the describe block in the previous line, or this test block needed to change at all